x-pack/plugins/cloud_security_posture/public/pages/findings/findings_flyout/resource_tab.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  EuiAccordion,
+  EuiCode,
+  EuiCodeBlock,
+  EuiDescriptionList,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  useEuiTheme,
+} from '@elastic/eui';
+import { CspFinding } from '@kbn/cloud-security-posture-plugin/public/pages/findings/types';
+import React, { useMemo } from 'react';
+import * as TEXT from '@kbn/cloud-security-posture-plugin/public/pages/findings/translations';
+import { getFlattenedObject } from '@kbn/std';
+
+const getDescriptionDisplay = (value: any) => {
+  if (typeof value === 'boolean' || value === undefined || value === null) {
+    return <EuiCode>{JSON.stringify(value)}</EuiCode>;
```

Comment:
`JSON.stringify(undefined)` results in an `undefined` value, not a string, I believe this is not what you meant to happen